docs/development/local_setup.md

```diff
@@ -0,0 +1,82 @@
+# Preparing the Local Development Setup (Mac OS X)
+
+Conceptionally, the Node Controller Manager is designated to run in a container within a Pod inside a Kubernetes cluster. However, if you want to develop it, you may want to work locally with the Node Controller Manager without building a Docker image and deploying it to a cluster each and every time. That means that the Node Controller Manager runs outside a Kubernetes cluster which requires providing a [Kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig/) in your local filesystem and point the Node Controller Manager to it when running it (see below). 
+
+Although the following installation instructions are for Mac OS X, similar alternate commands could be found for any other Linux distributions.
```

Comment:
I guess you meant (Mac OS X is not a Linux):
"for any Linux distribution"

Reply:
Done.